archive/Thematic/2021/Februrary/progressed/nba/content.md

```diff
@@ -1,20 +1,17 @@
+The Washington Post’s [Ben Golliver](https://benjamingolliver.com/) joins Hacker Noon CEO [David Smooke](https://www.davidsmooke.net/) to discuss the process of writing his new book, [“Bubbleball: Inside the NBA’s Fight to Save a Season,”](https://www.amazon.com/Bubbleball-Inside-NBAs-Fight-Season/dp/1419755536?&linkCode=sl1&tag=artmapincdbah-20&linkId=d97955373324ba2efdf60c7550df4c5f&language=en_US&ref_=as_li_ss_tl) and the pandemic’s impact on the NBA’s revenue. With no/few fans in the stands and the decline in the number of games from the pandemic, the National Basketball Association is expected to loss about $4 billion dollars in 2021. In addition to discussing his bubble writing and publishing process, we talk about the NBA’s options to stimulate revenue, such as [expansion fees for new franchises](https://www.washingtonpost.com/sports/2021/01/08/nba-expansion-seattle-las-vegas/) and how to make substantial investments into the NBA’s digital experience.
 
-Some [related reads on Hacker Noon](http://me.dm/r-ASKbSBXuIR?source=email-4a4) (where we usually publish more about things like [software development](http://me.dm/r-0jw2WCz-cT), [blockchains](http://me.dm/r-PGO2fN_ukq?source=email-494------227f4), [startups](http://me.dm/r-JP5EDNvEM3), and [everything tech](http://me.dm/r-ji_Wms6XpJ)):
+Some [related reads on Hacker Noon](https://hackernoon.com/) (where we usually publish more about things like [software development](https://hackernoon.com/tagged/software-development), [blockchains](https://hackernoon.com/tagged/blockchain), [startups](https://hackernoon.com/tagged/startup), and [everything tech](https://hackernoon.com/signup)):
 
-- [NBA’s Microsoft Partnership, Kawhi’s Logo Lawsuit, and the Modern Reporter’s Techstack](http://me.dm/r--NpWiZdjUW?source=.v3-4)
-- [Sports Teams Need To Embrace Digital Innovations to Regain Engagement and Revenue Lost to COVID-19](http://me.dm/r-X7sKBukHq3)
-- [The Gaming Ecosystem Explained](http://me.dm/r-pTlMU3ipDe)
-- [Making Up the Best Team in NBA History](http://me.dm/r-8W384qpmVp)
+* [NBA’s Microsoft Partnership, Kawhi’s Logo Lawsuit, and the Modern Reporter’s Techstack](https://podcast.hackernoon.com/e/washington-post-s-ben-golliver-discusses-nba-s-microsoft-partnership-kawhi-s-logo-lawsuit-and-the-modern-reporter-s-techstack/)
+* [Sports Teams Need To Embrace Digital Innovations to Regain Engagement and Revenue Lost to COVID-19](https://hackernoon.com/sports-teams-need-to-embrace-digital-innovations-to-regain-engagement-and-revenue-lost-to-covid-19-7yh310w)
+* [The Gaming Ecosystem Explained](https://hackernoon.com/the-gaming-ecosystem-explained-nk1d32ts)
+* [Making Up the Best Team in NBA History](https://hackernoon.com/making-up-the-best-team-in-nba-history-qfey32ei)
 
-**Buy the book — [Bubbleball: Inside the NBA’s Fight to Save a Season](http://me.dm/r-KivTMQYcQv)**
-
-HERE WE NEED TO ADD OUR LAST SENTENCE ABOUT PUBLISH ON HACKERNOON....
```

Comment:
this part is already added in the template by default so I just remove it. cc: @atherdon

Reply:
awesome